AirCasting/Services/UpdateSessionParams.swift

```diff
@@ -54,7 +54,7 @@ final class UpdateSessionParamsService {
             oldStream.thresholdVeryHigh = streamOutput.threshold_very_high
             oldStream.gotDeleted = streamOutput.deleted ?? false
             
-            let existingThreshold: SensorThreshold? = try context.existingObject(sensorName: streamOutput.sensor_name)
+            let existingThreshold: SensorThreshold? = try context.existingObject(sensorName: "AirBeam3-PM10")
```

Comment:
? Why ?

Reply:
I probably were sleeping when doing this 😅

Reply:
So you're coding while you sleep! This is how you make lots of views in such a short time ;D